example/cluster-machine-template-simple.yaml

```diff
@@ -0,0 +1,555 @@
+            ipProtocol: tcp
+# IpRange to authorize access to kubernetes endpoints (kube-apiserver), you must keep it and change it with a CIDR that best suits with your environment.
+            ipRange: "0.0.0.0/0"
+            fromPortRange: 6443
+            toPortRange: 6443
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: MachineDeployment
+metadata:
+  name: "hello-osc-md-0"
+  namespace: default
+spec:
+  clusterName: "hello-osc"
+  replicas: 1
+  selector:
+    matchLabels:
+  template:
+    spec:
+      clusterName: "hello-osc"
+      version: "1.24.0"
+      bootstrap:
+        configRef:
+          name: "hello-osc-md-0"
+          apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
+          kind: KubeadmConfigTemplate
+          namespace: default
+      infrastructureRef:
+        name: "hello-osc-md-0"
+        apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+        kind: OscMachineTemplate
+        namespace: default
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: OscMachineTemplate
+metadata:
+  name: "hello-osc-md-0"
+  namespace: default
+spec:
+  template:
+    spec:
+      node:
+        volumes:
+          - name: cluster-api-volume-kw
+            iops: 1000
+            size: 30
+            volumeType: io1
+            subregionName: eu-west-2a
+        vm:
+          name: cluster-api-vm-kw
+          imageId: ami-1fd1c79f
+          keypairName: cluster-api
+          volumeName: cluster-api-volume-kw
+          deviceName: /dev/xvdb
+          subnetName: cluster-api-subnet-kw
+          subregionName: eu-west-2a
+          securityGroupNames:
+            - name: cluster-api-securitygroups-kw
+          privateIps:
+            - name: cluster-api-kw-privateip-kw
+              privateIp: "10.0.0.138"
+          vmType: "tinav4.c2r4p2"
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: OscMachineTemplate
+metadata:
+  name: "hello-osc-control-plane"
+  namespace: default
+spec:
+  template:
+    spec:
+      node:
+        volumes:
+          - name: cluster-api-volume-kcp
+            iops: 1000
+            size: 30
+            volumeType: io1
+            subregionName: eu-west-2a
+        vm:
+          name: cluster-api-vm-kcp
+          imageId: ami-1fd1c79f
+          keypairName: cluster-api
+          volumeName: cluster-api-volume-kcp
+          deviceName: /dev/xvdb
+          subregionName: eu-west-2a
+          subnetName: cluster-api-subnet-kcp
+          loadBalancerName: osc-k8s
+          securityGroupNames:
+            - name: cluster-api-securitygroups-kcp
+          privateIps:
+            - name: cluster-api-privateip-kcp
+              privateIp: "10.0.0.38"
+          vmType: "tinav4.c2r4p2"
+---
+apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
+kind: KubeadmConfigTemplate
+metadata:
+  name: "hello-osc-md-0"
+  namespace: default
+spec:
+  template:
+    spec:
+      files:
+      - content: |
+          net.ipv4.conf.lxc*.rp_filter = 0
```

Comment:
Any links or topics related to this addition ?

Reply:
Replace and add in doc